src/jblockmain/Measurements.java

```diff
@@ -131,7 +137,7 @@ private void assignMeasurements(BufferedReader fileStream, boolean isTabbed)
                 while ((line = fileStream.readLine()) != null)
                 {
                     // Only process lines that start with a 1 and contain a []
-                    if (line.charAt(0) == '1' && line.contains("[") && line.contains("]"))
+                    if (line.length() > 0 && line.charAt(0) == '1' && line.contains("[") && line.contains("]"))
```

Comment:
I'm curious -- are we saying that `readline()` returns a non-null string of length 0 when it encounters a blank line?    

If this is true then this fix should be fine but I just wanted to check.

Reply:
So when the line.length() > 0 section on the if statement isn't included, we get a String index out of range exception. It's because when it encounters the first empty line, the one that was originally manually changed, it reads "" rather than null.